### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
# Description
Updated EN -> JA documentation translations

### CODE REVIEW

File: jekyll/_cci2_ja/server/installation/migrate-from-server-3-to-server-4.adoc [hunk 1]
```diff
@@ -154,17 +155,17 @@ helm diff upgrade <release-name> -n <namespace> -f <path>/kots-exporter/output/h
 * `distributor-*` のデプロイのシークレットや注釈が削除
 * アップストリームチャートの `postgresql` が更新
 * アップストリームチャートの再作成 (削除と作成)
-** Prometheus (circleci-server-kube-state-metrics, node-exporter, prometheus-server)
+** Prometheus (circleci-server-kube-state-metrics、node-exporter,prometheus-server)
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/server/installation/migrate-from-server-3-to-server-4.adoc:158]:
```suggestion
** Prometheus (circleci-server-kube-state-metrics、node-exporter、prometheus-server)
```

File: jekyll/_cci2_ja/server/installation/phase-3-execution-environments.adoc [hunk 2]
```diff
@@ -14,42 +14,42 @@ version:
 
 // This doc uses ifdef and ifndef directives to display or hide content specific to Google Cloud Storage (env-gcp) and AWS (env-aws). Currently, this affects only the generated PDFs. To ensure compatability with the Jekyll version, the directives test for logical opposites. For example, if the attribute is NOT env-aws, display this content. For more information, see https://docs.asciidoctor.org/asciidoc/latest/directives/ifdef-ifndef/.
 
-Before you begin with the CircleCI server v4.x execution installation phase, ensure you have run through link:/docs/server/installation/phase-1-prerequisites[Phase 1 – Prerequisites] and link:/docs/server/installation/phase-2-core-services[Phase 2 - Core Services Installation].
+CircleCI サーバー v4.x 実行環境のインストールステップを開始する前に、 link:/docs/ja/server/installation/phase-1-prerequisites[ステップ 1: 前提条件] と link:/docs/ja/server/installation/phase-2-core-services[ステップ 2: コアサービスのインストール] が実行済みであることを確認してください。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/server/installation/phase-3-execution-environments.adoc:17]: the earlier doc changes use `CircleCI Server` instead.
so i wonder, for consistency, should this stay as `CircleCI Server`, not `CircleCI サーバー` ?

```suggestion
CircleCI Server v4.x 実行環境のインストールステップを開始する前に、 link:/docs/ja/server/installation/phase-1-prerequisites[ステップ 1: 前提条件] と link:/docs/ja/server/installation/phase-2-core-services[ステップ 2: コアサービスのインストール] が実行済みであることを確認してください。
```

**Review by kelvintaywl — COMMENTED** [review]
LGTM besides 2 tiny suggestions 🙇

File: jekyll/_cci2_ja/server/installation/phase-3-execution-environments.adoc (hunk 2 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/server/installation/phase-3-execution-environments.adoc:17]: Oh thank you! `CircleCI Server` is correct.

**Review by kelvintaywl — APPROVED** [review]

**Pull request merged**